### PR TITLE
Using dynamic logger for GFX-H264 backend.

### PR DESCRIPTION
**Pull request by akallabeth**
* Use preallocated logger for GFX-H264 backend.
* Enabled OpenH264 traces on log level TRACE

### CODE REVIEW

File: libfreerdp/codec/h264_ffmpeg.c [hunk 1]
```diff
@@ -215,22 +215,22 @@ static int libavcodec_decompress(H264_CONTEXT* h264, const BYTE* pSrcData,
...
 	}
 
 #if 0
-	WLog_INFO(TAG,
-	          "libavcodec_decompress: frame decoded (status=%d, gotFrame=%d, width=%d, height=%d, Y=[%p,%d], U=[%p,%d], V=[%p,%d])",
-	          status, gotFrame, sys->videoFrame->width, sys->videoFrame->height,
-	          (void*) sys->videoFrame->data[0], sys->videoFrame->linesize[0],
-	          (void*) sys->videoFrame->data[1], sys->videoFrame->linesize[1],
-	          (void*) sys->videoFrame->data[2], sys->videoFrame->linesize[2]);
+	WLog_Print(h264->log, WLOG_INFO, (TAG,
```
Comment (mfleisz) [inline libfreerdp/codec/h264_ffmpeg.c]: The TAG stuff here looks wrong ...

**Comment by MartinHaimberger** [comment]
conflicts need to be resolved

**Comment by akallabeth** [comment]
@MartinHaimberger done

File: libfreerdp/codec/h264_mf.c [hunk 2]
```diff
@@ -182,7 +182,7 @@ static HRESULT mf_create_output_sample(H264_CONTEXT_MF* sys)
 
 	if (FAILED(hr))
 	{
-		WLog_ERR(TAG, "MFCreateSample failure: 0x%08"PRIX32"", hr);
+		WLog_Print(h264->log, WLOG_ERROR, "MFCreateSample failure: 0x%08"PRIX32"", hr);
```
Comment (mfleisz) [inline libfreerdp/codec/h264_mf.c]: looks like the context is named sys instead of h264 (that's why the VS builds fail)

**Comment by akallabeth** [comment]
@freerdp-bot test
@mfleisz ok, fixed (but the bot is still not happy)

**Comment by akallabeth** [comment]
Ok, rebased again, busy day today ^^

**Comment by akallabeth** [comment]
@mfleisz Ok, fixed and rebased the pull

**Pull request merged**